doc/kubernetes/modules/ROOT/pages/prerequisite/prerequisite-rosa.adoc

```diff
@@ -85,14 +85,14 @@ The above installation script creates an admin user automatically but in case th
 == Scaling the cluster's nodes on demand
 
 The standard setup of nodes might be too small for running a load test, at the same time using a different instance type and rebuilding the cluster takes a lot of time (about 45 minutes).
-To scale the cluster on demand, the standard setup has a machine pool named `scaling` with instances of type `m5.4xlarge` which is auto-scaled based on the current demand from 0 to 10 instances.
+To scale the cluster on demand, the standard setup has a machine pool named `scaling` with instances of type `m5.2xlarge` which is auto-scaled based on the current demand from 0 to 10 instances.
```

Comment:
Updated the number of instances. 
```suggestion
To scale the cluster on demand, the standard setup has a machine pool named `scaling` with instances of type `m5.2xlarge` which is auto-scaled based on the current demand from 4 to 15 instances.
```

Reply:
@kami619 I'm unsure if the minimum is 1 or 4 at this point. It looks like `rosa_create_cluster.sh` sets the minimum to 1.